diff --git a/node_modules/next/dist/client/script.js b/node_modules/next/dist/client/script.js
index 47d777f..73b6031 100644
--- a/node_modules/next/dist/client/script.js
+++ b/node_modules/next/dist/client/script.js
@@ -133,6 +133,9 @@ const loadScript = (props)=>{
         }
         const attr = _headmanager.DOMAttributeNames[k] || k.toLowerCase();
         el.setAttribute(attr, value);
+        if (value === false) {
+            el.removeAttribute(attr);
+        }
     }
     if (strategy === "worker") {
         el.setAttribute("type", "text/partytown");
